lib/pages/dashboard/dashboard_club_teams.dart

```diff
@@ -94,4 +95,8 @@ class _DashboardClubTeamsState extends State<DashboardClubTeams> with SingleTick
     );
   }
 
+  @override
+  // TODO: implement wantKeepAlive
```

Comment:
To remove?
If you think, this method has to be implemented create an issue and explain why and how this can be implemented, but I think your current implementation (always return true) is enough

Reply:
Yeah that's just an oversight. It's an autogenerated method !